ci-operator/step-registry/distributed-tracing/tests/tracing-ui/integration/distributed-tracing-tests-tracing-ui-integration-commands.sh

```diff
@@ -0,0 +1,148 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# List of variables to check.
+vars=(
+  CYPRESS_SKIP_COO_INSTALL
+  CYPRESS_COO_UI_INSTALL
+  CYPRESS_KONFLUX_COO_BUNDLE_IMAGE
+  CYPRESS_CUSTOM_COO_BUNDLE_IMAGE
+  CYPRESS_DT_CONSOLE_IMAGE
+)
+
+# Loop through each variable.
+for var in "${vars[@]}"; do
+  if [[ -z "${!var}" ]]; then
+    unset "$var"
+    echo "Unset variable: $var"
+  else
+    echo "$var is set to '${!var}'"
+  fi
+done
```

Comment:
I found nowhere there vars are used in the step shell, seems currently we can't testing by using bundle image

Reply:
This step you are referencing just unsets the vars which don't have any value. In the job logs, you can see this.
```

CYPRESS_SKIP_COO_INSTALL is set to 'true'
Unset variable: CYPRESS_COO_UI_INSTALL
Unset variable: CYPRESS_KONFLUX_COO_BUNDLE_IMAGE
Unset variable: CYPRESS_CUSTOM_COO_BUNDLE_IMAGE
Unset variable: CYPRESS_DT_CONSOLE_IMAGE
```

Since CYPRESS_SKIP_COO_INSTALL has a value true, it wont be unset.

As for using Konflux bundle image, to use the bundle, we set CYPRESS_KONFLUX_COO_BUNDLE_IMAGE with the COO bundle image. Once set, COO will be installed from the Cypress test run using the bundle and rest of the operators will be installed from redhat-operators catalog source required by test run.

Reply:
The CYPRESS_KONFLUX_COO_BUNDLE_IMAGE can either be set directly from step env vars or the job env vars.

Reply:
You can check the Cypress spec from the tracing console repo on how the vars are used. https://github.com/openshift/distributed-tracing-console-plugin/blob/main/tests/tests/dt-plugin-tests.cy.ts#L60